File: myfile.rb

class Grader
    def converter
        puts "What is you name?"
        studentName = gets
        puts studentName + "what is the title of the assignment?"
        assignment = gets
        puts "What grade did you get on assignment"
        gradeStr = gets
        studentGrades = gradeStr.to_i
        grades =["A","B","C","D","F"]
        if studentGrades >= 90
            puts studentName + " You got an " + grades[0]
        elsif studentGrades >= 80
            puts studentName + " You got an " + grades[1]
        elsif studentGrades >= 70
            puts studentName + " You got an " + grades[2]
        elsif studentGrades >= 60
            puts studentName + " You got an " + grades[3]
        else studentGrades
            puts studentName + " You got an " + grades[4]
        end
    end        
end
grade = Grader.new
grade.converter

File: myfile.js
const readline = require("readline");
const rl =readline.createInterface({
    input:process.stdin,
    output:process.stdout
})

class Grader{
    converter(){
        rl.question("What is your name?",(studentName)=>{
            rl.question("What is the title of the assignment?",(assignmentGrade)=>{
                rl.question("what grade did you get?", (studentGrades)=>{
                    grades=["A","B","C","D","F"]
                    if (studentGrades >= 90){
                        console.log(studentName + " you got an " + grades[0])
                    }
                    else if (studentGrades >= 80){
                        console.log(studentName + " you got an " + grades[1])
                    }
                    else if (studentGrades >= 70){
                        console.log(studentName + " you got an " + grades[2])
                    }
                    else if (studentGrades >= 60){
                        console.log(studentName + " you got an " + grades[3])
                    }
                    else{
                        console.log(studentName + " you got an " + grades[4])
                    }
                    rl.close()
                })
            })
    })
    }
}
grade = new Grader();
grade.converter();